doc/source/examples/quickstart.ipynb

```diff
@@ -1513,7 +2604,1416 @@
+      "image/svg+xml": [
```

Comment:
See above.

Reply:
Have you set your matplotlib to use the svg backend? I'm also not sure this is a bad thing, could make diffing output easier

Reply:
I just created a fresh `conda` environment from the file provided. I was working on macOS 10.15 if that matters.

It might be making diffing easier, but it adds a lot of lines.